src/setfit/onnx.py

```diff
@@ -0,0 +1,227 @@
+from .modeling import SetFitModel
+from sentence_transformers import models
+import numpy as np
+import onnx
+import warnings
+from typing import Optional, Union
+import torch
+from sklearn.linear_model import LogisticRegression
+
+
+class ONNXSetFitModel(torch.nn.Module):
+    def __init__(
+        self,
+        model_body,
+        pooler,
```

Comment:
I guess this can be a generic `Callable` right?

Reply:
I believe the type of the pooler is `sentence_transformers.models.Pooling.Pooling` which is a `nn.Module`. However,  we grab the "sentence_embedding" output from the pooling layer so a generic nn.Module wouldn't work.  What are your thoughts? Should we say the type of Pooler is Pooling or nn.Module and add some caveats in the docstring?

Reply:
Let me know if my solution addresses this comment : ).